src/jaxsim/api/ode.py

```diff
@@ -198,7 +202,7 @@ def system_velocity_dynamics(
 
 @jax.jit
 def system_position_dynamics(
-    model: Model.JaxSimModel, data: Data.JaxSimModelData
+    model: js.model.JaxSimModel, data: js.data.JaxSimModelData
```

Comment:
`model` is not used
```suggestion
    data: js.data.JaxSimModelData
```

Reply:
Waiting the semi-implicit integrators, this is intended. I'll leave it as it is in the `functional` branch, let's discuss it again before merging into `main`.